dockerfile

```diff
@@ -2,28 +2,30 @@
 # docker build -t ohif/viewer:latest .
 FROM node:11.2.0-slim as builder
 
-RUN apt-get update && apt-get install -y git yarn
+# RUN apt-get update && apt-get install -y git yarn
```

Comment:
Does running this cause things to break? Or is this a speed optimization? Any advantage in using a more up-to-date image?

Reply:
I forgot to uncomment that part. Thank you!